src/ngraph/runtime/gpu/unit_test.manifest

```diff
@@ -258,6 +258,13 @@ model_split_equal_parts_default
 model_split_equal_parts_2d
 model_split_variable_parts_2d
 model_expand_static_shape
+expand_1_dyn_shape
+expand_2_dyn_shape
+expand_3_dyn_shape
+expand_4_dyn_shape
+expand_5_dyn_shape
+expand_6_dyn_shape
+expand_uint16_dyn_shape
```

Comment:
I'm surprised that these tests does not work on GPU.
`Expand` op use only `Constant` and `Multiply` ops could you explain it?

Reply:
On GPU tests fail as following:
```
[ RUN      ] onnx_dyn_shapes_GPU.expand_2d_1_dyn_shape
/home/dockuser/ngraph/test/util/test_case.cpp:76: Failure
Value of: values_match(expected_result_constant, result_tensor)
  Actual: false (1 is not close to 2 at index 1
1 is not close to 3 at index 2
1 is not close to 0 at index 3
1 is not close to 0 at index 4
1 is not close to 0 at index 5
diff count: 35 out of 36
```